tket/src/Utils/UnitID.cpp

```diff
@@ -42,6 +42,13 @@ void from_json(const nlohmann::json& j, Bit& cb) { json_to_unitid(j, cb); }
 void to_json(nlohmann::json& j, const Node& node) { unitid_to_json(j, node); }
 void from_json(const nlohmann::json& j, Node& node) { json_to_unitid(j, node); }
 
+void to_json(nlohmann::json&, const WASMUID&) {
+  throw std::logic_error("wasm to json conversion");
+}
+void from_json(const nlohmann::json&, WASMUID&) {
+  throw std::logic_error("json to wasm conversion");
+}
```

Comment:
My plan was to hide it from the public and to have the functionality to automatically generate them when adding wasm ops. If we want to show them to the public this would need to be changed.